packages/desktop-client/src/components/schedules/index.js

```diff
@@ -10,6 +10,7 @@ import { Page } from '../Page';
 import { SchedulesTable, ROW_HEIGHT } from './SchedulesTable';
 
 export default function Schedules() {
+  console.log(`<Schedules> mounted at ${new Date().toISOString()}`);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should this be removed?